jira-feature-collector/src/main/java/com/capitalone/dashboard/datafactory/jira/JiraDataFactoryImpl.java

```diff
@@ -145,12 +162,29 @@ public JiraDataFactoryImpl(int inPageSize, String jiraCredentials, String jiraBa
 	 */
 	public JiraDataFactoryImpl(int inPageSize, String jiraCredentials, String jiraBaseUrl,
 			String jiraProxyUrl, String jiraProxyPort) {
-		URI jiraUri = this.createJiraConnection(jiraBaseUrl, jiraProxyUrl + ":" + jiraProxyPort,
-				this.decodeCredentials(jiraCredentials).get("username"),
-				this.decodeCredentials(jiraCredentials).get("password"));
-		this.client = factory.createWithBasicHttpAuthentication(jiraUri,
-				this.decodeCredentials(jiraCredentials).get("username"),
-				this.decodeCredentials(jiraCredentials).get("password"));
+		URI jiraUri;
+		if ((jiraBaseUrl != null) && (!jiraBaseUrl.isEmpty())) {
```

Comment:
can just Strings.isNullOrEmpty()  - we're pulling in guava, might as well use it..